tests/test/util.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import pyTigerGraph as tg
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def get_featurizer():
+    graph_name = "graph_algorithms_testing"
+
+    host_name = os.getenv("HOST_NAME")
+    user_name = os.getenv("USER_NAME")
+    password = os.getenv("PASS")
+    conn = tg.TigerGraphConnection(
+        host=host_name,
+        username=user_name,
+        password=password,
+        graphname=graph_name,
+        tgCloud=True,
+    )
+    conn.getToken()
```

Comment:
Potentially parameterize if a user needs to get a token or not? Would be handy for us if we run the test suite against a Zulip machine (which has no token auth enabled by default)